src/core/server/elasticsearch/version_check/ensure_es_version.ts

```diff
@@ -112,16 +119,24 @@ export function mapNodesVersionCompatibility(
     kibanaVersion,
   };
 }
-
+// Returns true if NodesVersionCompatibility nodesInfoRequestError is the same
+function compareNodesInfoErrorMessages(
+  prev: NodesVersionCompatibility,
+  curr: NodesVersionCompatibility
+): boolean {
+  return prev.nodesInfoRequestError?.message === curr.nodesInfoRequestError?.message;
+}
 // Returns true if two NodesVersionCompatibility entries match
 function compareNodes(prev: NodesVersionCompatibility, curr: NodesVersionCompatibility) {
   const nodesEqual = (n: NodeInfo, m: NodeInfo) => n.ip === m.ip && n.version === m.version;
+  // if we pass the full error back in here, we can check to see if it's the same error rather than checking
```

Comment:
I think these are dev notes and can be removed now, can they? :)

Reply:
Yes they can.